### PR TITLE
0.8.7 Release announcement

### PR DESCRIPTION
**Pull request by hrkrshnn**

### CODE REVIEW

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+layout: post
+published: true
+title: 'Solidity 0.8.7 Release Announcement'
+date: '2021-08-11'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.7](https://github.com/ethereum/solidity/releases/tag/v0.8.7) introduces support for
+[London Hardfork](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/), includes
```
Comment (franzihei) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:11]:
```suggestion
the [London Hardfork](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/), includes
```

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+choose the precise encoding for division and modulo. This option is expected to
+be useful only when employing other solvers than z3.
+
+Lastly, underflow and overflow targets are not checked by default anymore.
+This follows the broad use of compiler generated checks which are expected to
+happen in Solidity >=0.8.0. Underflow and overflow checks can still be performed
+by selecting them via the command line option ``--model-checker-targets`` or
+the JSON option ``settings.modelChecker.targets``.
+
+Please see the SMTChecker documentation in the Solidity docs for more detailed
```
Comment (franzihei) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:57]:
```suggestion
Please see the [SMTChecker documentation](https://docs.soliditylang.org/en/latest/smtchecker.html) in the Solidity docs for more detailed
```

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+ * Code Generator: Fix internal compiler error when calling functions bound to calldata structs and arrays.
+ * Code Generator: Fix internal compiler error when passing a 32-byte hex literal or a zero literal to ``bytes.concat()`` by disallowing such literals.
+ * Commandline Interface: Apply ``--optimizer-runs`` option in assembly / yul mode.
+ * Commandline Interface: Fix crash when a directory path is passed to ``--standard-json``.
+ * Commandline Interface: Read JSON from standard input when ``--standard-json`` gets ``-`` as a file name.
+ * Standard JSON: Include source location for errors in files with empty name.
+ * Type Checker: Fix internal error and prevent static calls to unimplemented modifiers.
+ * Yul Code Generator: Fix internal compiler error when using a long literal with bitwise negation.
+ * Yul Code Generator: Fix source location references for calls to builtin functions.
+ * Yul Parser: Fix source location references for ``if`` statements.
```
Comment (franzihei) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:90]:
```suggestion
 * Yul Parser: Fix source location references for ``if`` statements.
 
 A big thank you to all contributors who helped make this release possible!

Download the new version of Solidity [here](https://github.com/ethereum/solidity/releases/tag/v0.8.7).
```

**Review by franzihei — COMMENTED** [review]
A few small things

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+layout: post
+published: true
+title: 'Solidity 0.8.7 Release Announcement'
+date: '2021-08-11'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.7](https://github.com/ethereum/solidity/releases/tag/v0.8.7) introduces support for
+the [London Hardfork](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/), includes
```
Comment (chriseth) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:11]: The blog post uses the term "London upgrade" - not sure if that is important for us.

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+various improvements to Yul to EVM code transformation, the SMTChecker and some bugfixes.
+
+## Support for London Hardfork
+
+Solidity adds support to the `BASEFEE` opcode ([EIP-3198](https://eips.ethereum.org/EIPS/eip-3198)
```
Comment (chriseth) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:16]:
```suggestion
Solidity adds support for the `BASEFEE` opcode ([EIP-3198](https://eips.ethereum.org/EIPS/eip-3198)
```

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md (hunk 4 above)
Comment (hrkrshnn) [reply _posts/2021-08-11-solidity-0.8.7-release-announcement.md:11]: Yeah, was wondering if the word "hardfork" should be avoided.

Comment (hrkrshnn) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:11]:
```suggestion
the [London upgrade](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/), includes
```

Comment (hrkrshnn) [reply _posts/2021-08-11-solidity-0.8.7-release-announcement.md]: fixed.

File: _posts/2021-08-11-solidity-0.8.7-release-announcement.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+date: '2021-08-11'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.7](https://github.com/ethereum/solidity/releases/tag/v0.8.7) introduces support for
+the [London upgrade](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/), includes
+various improvements to Yul to EVM code transformation, the SMTChecker and some bugfixes.
+
+## Support for London Hardfork
```
Comment (hrkrshnn) [inline _posts/2021-08-11-solidity-0.8.7-release-announcement.md:14]:
```suggestion
## Support for London Upgrade
```

**Review by franzihei — APPROVED** [review]

**Pull request merged**